selftests/functional/test_interrupt.py

```diff
@@ -121,6 +121,9 @@ def wait_until_no_badtest():
         # Make sure the Killing test subprocess message did appear
         self.assertIn('Killing test subprocess', proc.get_output())
 
+    @unittest.skipIf(int(os.environ.get("AVOCADO_CHECK_LEVEL", 0)) < 2,
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
I know you can only filter `avocado.Test`s, that's why I said " But this involves making those avocado.Test, and not unittest.TestCase".

My point is that adding if I were to invest in more code, I'd invest in making our tests Avocado Instrumented tests, and reuse the already presenta test tags, instead of adding decorators for levels on unittest.TestCase.  But that's a completely different discussion.